auto_landing/src/trajectory_generation_tracker_node.cpp

```diff
@@ -0,0 +1,25 @@
+#include <auto_landing/trajectory_generation_tracker.hpp>
+
+using namespace ariitk::auto_landing;
+
+int main(int argc, char** argv) {
+    ros::init(argc, argv, "trajectory_pub");
+    TrajectoryGenerationTracking tracking(argv);
+    
+    ros::Rate loop_rate(10);
+
+    int count = 0;
+    while (count<10) {
+        ros::spinOnce();
```

Comment:
what's the purpose of this code block?

Reply:
It is to ensure that the Odometry callbacks have been run atleast once.